actors/builtin/storage_power/policy.go

```diff
@@ -1,26 +1,27 @@
 package storage_power
 
 import (
+	big "github.com/filecoin-project/specs-actors/actors/abi/big"
 	abi "github.com/filecoin-project/specs-actors/actors/abi"
 	builtin "github.com/filecoin-project/specs-actors/actors/builtin"
-	autil "github.com/filecoin-project/specs-actors/actors/util"
+	indices "github.com/filecoin-project/specs-actors/actors/runtime/indices"
 )
 
 // TerminationFault
 func pledgePenaltyForSectorTermination(weight SectorStorageWeightDesc, termType builtin.SectorTermination) abi.TokenAmount {
-	autil.PARAM_FINISH()
+	// autil.PARAM_FINISH()
```

Comment:
Remove the `autil` and `()` from there everywhere. We can just use the token `PARAM_FINISH` in a comment. to mark these places